app/assets/stylesheets/custom.scss

```diff
@@ -0,0 +1,103 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+/* mixins, variables, etc. */
+
+$gray-medium-light: #eaeaea;
```

Comment:
Tạo file color.scss, chuyển toàn bộ biến màu vào file color.scss rồi import vào file này